### ISSUE TITLE
Assignment 15

### ISSUE THREAD

**Issue by FSGabrsek** [FSGabrsek/AE-I_Intro-Data-Science#21]
### Decision tree

- [x] Split the penguins dataset into a training and test set
- [x] Fit a DecisionTreeClassifier with a valid training data
- [x] Document the accuracy of the tree for the chosen set size and depth
